spec/calendar.html

```diff
@@ -752,18 +736,18 @@ <h1>Temporal.Calendar.prototype.yearMonthFromFields ( _fields_, _options_, _cons
         1. If Type(_fields_) is not Object, throw a *TypeError* exception.
         1. Set _options_ to ? NormalizeOptionsObject(_options_).
         1. Let _result_ be ? ISOYearMonthFromFields(_fields_, _options_).
-        1. Return ? CreateTemporalYearMonthFromStatic(_constructor_, _result_.[[Year]], _result_.[[Month]], _calendar_, _result_.[[ReferenceISODay]]).
+        1. Return ? CreateTemporalYearMonth(_result_.[[Year]], _result_.[[Month]], _calendar_, _result_.[[ReferenceISODay]]).
       </emu-alg>
     </emu-clause>
 
     <emu-clause id="sec-temporal.calendar.prototype.monthdayfromfields">
-      <h1>Temporal.Calendar.prototype.monthDayFromFields ( _fields_, _options_, _constructor_ )</h1>
+      <h1>Temporal.Calendar.prototype.monthDayFromFields ( _fields_, _options_ )</h1>
       <p>
         An ECMAScript implementation that includes the ECMA-402 Internationalization API must implement the `monthDayFromFields` method as specified in the ECMA-402 specification.
         If an ECMAScript implementation does not include the ECMA-402 API the following specification of the `monthDayFromFields` method is used.
       </p>
       <p>
-        The `monthDayFromFields` method takes three arguments, _fields_, _options_, and _constructor_.
+        The `monthDayFromFields` method takes two arguments, _fields_ and _options_.
```

Comment:
i think we're generally removing the count and enumeration of arguments here (since it's redundant with the method signature)?

Reply:
Good! I did find that unnecessary. Is this something that's worth opening a ticket to remove it from the rest of the spec text?

Reply:
Already done: https://github.com/tc39/ecma262/issues/2304

Reply:
Sorry, should've specified, the rest of the Temporal spec text (other than these four methods being changed here). Sounds like yes :smile: